lib/Database.rb

```diff
@@ -0,0 +1,14 @@
+require "sequel"
+
+module Database
+    DB = Sequel.connect("sqlite://development.db")
+    
+    unless DB.table_exists?(:tasks)
```

Comment:
Same thing I pointed out with Main: this is code is automatically run—which is a bad idea.